reference-architecture/vmware-ansible/roles/docker-storage-setup/tasks/main.yaml

```diff
@@ -0,0 +1,13 @@
+---
+- name: create the docker-storage-setup config file
+  copy:
+     src: docker-storage-setup
+     dest: /etc/sysconfig/docker-storage-setup
```

Comment:
I'm thinking more for setting options like autoextend threshold, autoextend percent, etc.